middleware/index.ts

```diff
@@ -305,25 +294,21 @@ function createLogger(requestID?: string): Logger {
 
 const userFn = getUserFn();
 
-export default async function systemFn(message: object): Promise<object> {
+export default async function systemFn(message: any): Promise<any> {
```

Comment:
I wonder if we should do an ad-hoc type `{ headers:... }`. I don't really like the usage of `any`